Italian/main/Settings.apk/res/values-it/strings.xml

```diff
@@ -3955,22 +3955,22 @@ ed esegui il software."</string>
     <string name="filter_apps_cached">Nella cache</string>
     <string name="screen">Schermo</string>
     <string name="system">Sistema</string>
-    <string name="screen_color_temperature">Temperatura colori</string>
+    <string name="screen_color_temperature">Colori</string>
```

Comment:
"Temperatura di colori" è praticamente una sinestesia sulla sinestesia, tant'è che si dice "il colore è caldo" e mai "la temperatura del colore è X", per questo ho preferito cambiare.
Una versione alternativa (e probabilmente migliore ora che ho approfondito) può essere "Temperatura di colore" o "Temperatura colore", che è una cosa ben definita e già usata come stringa per esempio in ubuntu nelle impostazioni dello schermo